### ISSUE TITLE
Challenges with norms in computed measures

### ISSUE THREAD

**Issue by gdevenyi** [DouglasNeuroInformatics/OpenDataCapture#975]
I realized from the conversation yesterday with our visitors, that if we want to use tabular norms, some of them are indexed to age/sex, which may not be directly present in a given form.

**Comment by joshunrau** [comment]
Right, so you want to be able to compute measures based on the subject?

**Comment by gdevenyi** [comment]
Correct many psych tests are age and sex normed so the standardized score depends on the specific participant

**Comment by joshunrau** [comment]
Makes sense. Let me see what I can do
